content/en/docs/security/_index.md

```diff
@@ -1,4 +1,19 @@
 ---
 title: Security
-weight: 150
+description: How to use OpenTelemetry Collector securely.
```

Comment:
```suggestion
description: How to use OpenTelemetry Collector securely.
```

Not being a fan of adverbs in titles aside, I think the section could use a more noun-based title. For example:

- OpenTelemetry security
- Security in OpenTelemetry
- Secure the OpenTelemetry Collector

Etc.

Reply:
Incorporated. Thank you!